content/gherkin/reference.md

```diff
@@ -330,6 +331,60 @@ to match the step against a step definition.
 
 You can also use parameters in [multiline step arguments](#step-arguments).
 
+## Rule
+
+The `Rule` entity is a rather new idiom in Gherkin (since Gherkin v6).
+Each `Rule` is intended to represent one *business rule* that should be implemented.
+A `Rule` is used to group together several `Example` (aka: `Scenario`) / `Scenario Outline`
+that belong to this *business rule*. In addition, a `Rule` may contain a `Background` section.
+
+Rule parent:
+
+- `Feature` (cardinality: one)
+
+Rule children:
+
+- `Description` (cardinality: optional, `0 .. 1`)
+- `Background` (cardinality: optional, `0 .. 1`)
+- `Example`/`Scenario` (cardinality: many, `0 .. N`)
+- `Scenario Outline`/`Scenario Template` (cardinality: many, `0 .. N`)
+
+Rule constraints:
+
+- A `Rule` should have at least one child (that is neither `Description` nor `Background`).
+
+Rule supports:
```

Comment:
I like how you add these headers (children, contraints, etc)! I'm not sure this particular one adds value though; as this information is also provided elsewhere afaik. (Although maybe check that Rule is mentioned under Tags). 
Now I'm wondering if we should have similar headers for all the other keywords...?
I also like that you put an example!